03-Custom-Actions-In-Python/02-An-Initial-Custom-Action.ipynb

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
<p>1 - From how I understand currently, an operation is just an action that is ready to be added (after being wrapped in CustomOperation object), so it's confusing to me to say "custom actions and operations do" here.</p><p>1.1 - Add something saying "a custom operation is a custom action + information about when to use the action in the form of a hoomd.trigger"</p><p> </p><p>2 Suggest re-wording last sentence:</p><p>"To use a custom operation, we must add it to a <code>hoomd.Simulation</code> object."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>